core/db/migrate/20221229132350_create_spree_google_export_options.rb

```diff
@@ -0,0 +1,16 @@
+class CreateSpreeGoogleExportOptions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :spree_google_export_options do |t|
```

Comment:
I wonder if it would be more accurate to name the table/resource `google_feed_options` instead of `google_export_options`? 🤔  when I think about export options, what comes to mind are things like export format (csv, etc), what data is being exported, and things like that. Whereas "google feed" would make it clear that we're talking about options related to how the feed is generated.

Something to consider. (@przemosk  - any thoughts?)

Reply:
yeah, table name could be more accurate, I think I can propose `google_feed_settings`,  to stick with table convention in Spree I would suggest to add suffix `spree_google_feed_settings` wdyt @nciemniak @wjwitek ?

Reply:
that would definetly by clearer than current naming

Reply:
@przemosk yeah, continuity is great. +1 for naming it settings.